documentation/airport-guides/kabq.md

```diff
@@ -0,0 +1,62 @@
+### About this facility
+Albuquerque Combined TRACON and ATCT (ABQ), Albuquerque, New Mexico, USA  
+198,987 annual operations in 2017 ([FAA ATADS](https://aspm.faa.gov/opsnet/sys/Tracon.asp))
+
+This level 8 combined TRACON/ATCT facility controls a number of airfields in Albuquerque and its surrounding areas. Since this is a combined facility, the TRACON and ATC Tower are co-located at Albuquerque International Sunport (KABQ), just over 3 miles from Downtown Albuquerque.
+
+### Airspace
+The ABQ TRACON airspace encompasses a fairly large area around Albuquerque extending from the surface up to 20,000 feet (FL200). Class C airspace exists with tops of 9,400 feet over Albuquerque.
+
+### Instrument Procedures
+Most procedures for KABQ are RNAV, so controllers only need to vector arriving aircraft to their final approach course. Different RNAV departures will be used depending on the runway configuration:  
+
+| RWYs 21/26 | RWYs 3/8 |
+|------------|----------|
+| ADYOS3     | GRZZZ3   |
+| ATOMK3     | JEMEZ3   |
+| BOSQE3     | MNZNO3   |
+| DOOKK3     | RDRNR3   |
+| FYSTA3     |          |  
+
+Non-RNAV departures apply to all runways
+
+### Sectors Available in OpenScope
+```
+N - North Approach
+S - South Approach
+F - Final Approach
+```
+The airspace available in Openscope is a combination of these 3 sectors.
+
+### Airports of Interest
+
+#### KABQ (Primary Airport)
+Albuquerque International Sunport  
+134,992 annual operations
+
+Albuquerque Sunport is the largest commercial airport in the state of New Mexico, handling almost 5 million passengers anually. Since 1940, the airport shares its three runways with Kirtland Air Force Base, which also handles rescue and firefighting operations. Runway 3/21 is the primary runway for both departures and arrivals, although runway 8/26 can also be used in heavy traffic conditions. Due to high terrain to the East of the airfield, ILS approaches are not possible into runways 21 and 26, so RNAV approaches are normally used instead.
+
+#### KAEG
+Double Eagle II Airport  
+63,843 annual operations  
+11.2nm Northwest of KABQ
+
+Named after the Double Eagle II balloon, the first balloon to cross the Atlantic Ocean, this airport is located 7 miles Northwest of Downtown Albuquerque. It used primarily for general and corporate aviation.
+
+#### KE98
+Mid Valley Airpark  
+20,000 annual operations  
+18.0nm Southwest of KABQ
+
+Mid Valley Airpark is a fly-in community located in Los Lunas, containing its own privately owned airport which has been accessible to the public since 1970. It houses aproximately 200 aircarft used by the residents of the community. All roads within the Airpark also serve as Taxiways, called "Auto-Plane Drives", aircraft have the right of way.

```

Comment:
typos:
aproximately > approximately
aircarft > aircraft

Taxiways should be lowercase
Semicolon instead of comma before ‘aircraft have right of way.’